src/Entities/Boat.h

```diff
@@ -13,14 +13,26 @@
 
 
 
-
+// tolua_begin
 
 class cBoat :
 	public cEntity
 {
 	typedef cEntity super;
 
 public:
+	enum eBoatType
```

Comment:
I think the `Boat` part of the name is superficial. Compare: `cBoat::eBoatType` vs `cBoat::eType`. (but keep the `bt` prefix on values)